src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/boss_urok_doomhowl.cpp

```diff
@@ -47,14 +47,19 @@ class boss_urok_doomhowl : public CreatureScript
         {
             me->CastSpell(me, SPELL_UROK_SPAWN, true);
             BossAI::InitializeAI();
+            Talk(SAY_SUMMON);
+            if (Unit* target = me->SelectNearestTarget(500))
+            {
+                me->AI()->AttackStart(target);
+            }
         }
 
         void EnterCombat(Unit* /*who*/) override
         {
             _EnterCombat();
             events.ScheduleEvent(SPELL_REND, urand(17000, 20000));
             events.ScheduleEvent(SPELL_STRIKE, urand(10000, 12000));
-            Talk(SAY_AGGRO);
```

Comment:
why was this removed?

Reply:
if it is the wrong yell, you need to fix it in creature_text instead of just deleting it

Reply:
My understanding is that he has no on_combat yell. No boss will start a fight with "me smash you die" like he's a random ogre. And his yell is "You have **summoned** me, fool! Now Die!". See here https://youtu.be/jeTFFQ37tW0?t=8  
So I put that yell on the on_summon, and since I understood it that he should engage on summon, then there should be no on_combat?  It's true that he might say something when he is engaged after a reset, but I don't know if we have any source on that? Or does he despawn on wipe?

Here they suggest he does a yell and a say in sequence, basically erasing the yell. https://youtu.be/W7YdNcfEDgo?t=204 That makes no sense and can only be an error.

Reply:
I'd put the other text in EnterCombat instead of OnSummon just incase of resets etc

Reply:
The yell specifically mentions summoning. Ogres are stupid, but I feel that would be worse than having nothing. We can ask around and see if someone could run it on classic and vanish or something to see what happens

1. When he resets (despawn?)
2. on engage without summon (does he yell?)